luigi_pipeline/tests/model/test_seqr_mt_schema.py

```diff
@@ -71,13 +117,13 @@ def test_samples_num_alt(self):
         mt = seqr_schema.samples_no_call().samples_num_alt().select_annotated_mt()
         row = mt.rows().flatten().collect()[0]
         self.assertEqual(row.samples_no_call, set())
-        self.assertEqual(row['samples_num_alt.1'], {'NA19679', 'NA19675', 'NA20870', 'NA20876', 'NA20872'})
+        self.assertEqual(
+            row['samples_num_alt.1'], {'NA19679', 'NA19675', 'NA20870', 'NA20876', 'NA20872'}
+        )
         self.assertEqual(row['samples_num_alt.2'], set())
 
     def test_samples_gq(self):
-        non_empty = {
```

Comment:
If you leave a "magic trailing comma", `black` will preserve existing dictionary formatting.  Otherwise we get the greedy collapse.